sdetools/modules/import_veracode/veracode_integrator.py

```
@@ -11,15 +14,40 @@
 
 DEFAULT_MAPPING_FILE = os.path.join(media_path, 'veracode', 'sde_veracode_map.xml')
 
+class VeracodeAPI(RESTBase):
+    """ Base plugin for Veracode """
+
+    def __init__(self, config):
+        super(VeracodeAPI, self).__init__('vc', 'Veracode', config)
+
+    def parse_response(self, result, headers):
+        # Override to allow xml result
+        try:
+            result = minidom.parseString(result)
+        except Exception:
```

Catch only minidom exceptions here - let the rest propagate